internalstaging.datastage.io/portal/gitops.json

```diff
@@ -303,6 +317,34 @@
       "fieldMapping": [
         { "field": "consent_codes", "name": "Data Use Restriction" },
         { "field": "bmi", "name": "BMI" }
+        { "field": "cac_score", "name": "CAC Score" },
```

Comment:
i think youre missing a comma at the end of line 319 -- `{ "field": "bmi", "name": "BMI" },`

Reply:
comma added :)